content/en/docs/Overview/index.md

```diff
@@ -3,66 +3,72 @@ title: "Overview"
+eBPF (extended Berkeley Packet Filter) can run user's programs in the Linux kernel without modifying the kernel code or loading kernel modules. It is widely used in networking, security, monitoring and other relevant fields. Compared with iptables, Merbridge can shorten the data path between sidecars and services and therefore accelerate networking. Meanwhile, using Merbridge will not change the original architecture of Istio. The original logic is still valid. This means that if you don't want Merbridge anymore, just delete the DaemonSet. The original iptables will function again without any troubles.
 
 ## What Merbridge can do
 
 Merbridge has following core features:
 
 - Processing outbound traffic
 
-  Merbridge uses eBPF’s `connect` program to modify `user_ip` and `user_port`, so as to modify the destination address of a connection and ensure traffic can be sent to the new interface. In order to make Envoy identify the original destination address, the application (incl. Envoy) will call the `get_sockopt` function to obtain `ORIGINAL_DST` when receiving a connection.
+  Merbridge uses eBPF’s `connect` program to modify `user_ip` and `user_port`, so as to change the destination address of a connection and ensure traffic can be sent to the new interface. In order to help Envoy identify the original destination, the application (incl. Envoy) will call the `get_sockopt` function to get `ORIGINAL_DST` when receiving a connection.
 
 - Processing inbound traffic
 
-  The processing of inbound traffic is basically similar to outbound traffic. Note that eBPF cannot take effect in a specified namespace like iptables. The changes will be global. It means that if we use a Pod that is not originally managed by Istio, or an external IP address, serious problems will be encountered — like the connection not being established at all.
+  Inbound traffic is processed similarly to outbound traffic. Note that eBPF cannot take effect in a specified namespace like iptables, so changes will be global. It means that if we apply eBPF to Pods that are not originally managed by Istio, or an external IP, serious problems will occur, e.g., cannot establish a connection.
 
-  As a result, we designed a tiny control plane (deployed as a DaemonSet), which watches all pods — similar to the kubelet watching pods on the node — to write the pod IP addresses that have been injected into the sidecar to the `local_pod_ips` map. If the destination address is not in the map, Merbridge will not do anything to the original traffic.
+  To address this issue, we designed a tiny control plane, deployed as a DaemonSet. It can watch and get a list of all pods on the node, similar to kubelet. Then, Pod IPs injected into the sidecar will be written into the `local_pod_ips` map. For traffic with a destination address not in the map, Merbridge will not intercept it.
```

Comment:
```suggestion
  To address this issue, we designed a tiny control plane, deployed as a DaemonSet. It can help watch and get a list of all pods on the node, similar to kubelet. Then, Pod IPs injected into the sidecar will be written into the `local_pod_ips` map. For traffic with a destination address not in the map, Merbridge will not intercept it.
```